pages/common/git-show-ref.md

```diff
@@ -0,0 +1,20 @@
+# git show-ref
+
+> Git command for listing references.
+> More information: <https://git-scm.com/docs/git-show-ref>.
+
+- Show all refs in the repository:
+
+`git show-ref`
+
+- Show only heads references:
+
+`git show-ref --heads`
+
+- Show only tags references:
+
+`git show-ref --tags`
+
+- Verify that a certainr reference exists:
```

Comment:
typo?
```suggestion
- Verify that a given reference exists:
```

Reply:
Yea this is a typo, and i like given over certain, since the reference is indeed given through the arguments.